docs/development/codeguide.rst

```diff
@@ -190,20 +190,12 @@ Coding Style/Conventions
             See :doc:`codeguide_emacs` for some configuration options for Emacs
             that helps in ensuring conformance to PEP8.
 
-* Astropy source code should contain a comment at the beginning of the file (or
-  immediately after the ``#!/usr/bin env python`` command, if relevant)
+* Astropy source code should contain a comment at the beginning of the file
   pointing to the license for the Astropy source code.  This line should say::
 
       # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-* The following naming conventions::
-
-    import numpy as np
-    import matplotlib as mpl
-    import matplotlib.pyplot as plt
-    import pandas as pd
-
-  should be used wherever relevant. The complete list of conventional aliases can be found `here <https://docs.astral.sh/ruff/settings/#flake8-import-conventions-aliases>`_ . On the other hand::
```

Comment:
These conventions are already autofixed by ruff so there's no point in highlighting them in the dev guide.

Reply:
I agree. There is no point in cluttering the documentation with something that Ruff can enforce automatically.